core/dist/css/decanter.css

```diff
@@ -8472,9 +8477,18 @@ button,
     margin-top: 0;
     font-size: 1em;
     color: #4d4f53; }
+
+.su-hero--caption .su-hero__content-inner {
+  float: right; }
   @media only screen and (min-width: 992px) {
-    .su-hero--caption .su-hero__content {
-      padding-left: 40%; } }
+    .su-hero--caption .su-hero__content-inner {
+      width: 596px; } }
+  @media only screen and (min-width: 1200px) {
+    .su-hero--caption .su-hero__content-inner {
+      width: 720px; } }
+  @media only screen and (min-width: 1500px) {
+    .su-hero--caption .su-hero__content-inner {
+      width: 900px; } }
```

Comment:
Are we using fixed width container and not fluid 60%? I'm totally fine with this - looks close enough. Just want to double check because it's a little different from the request.